### ISSUE TITLE
Training on significantly different style: seeking tips

### ISSUE THREAD

**Issue by GShef** [hansam95/OSASIS#6]
Hello, 
I used your code to train_diffaeB.py to train on a significantly different style of images. I noticed that I can only supply 1 image at a time as style input for training. Given the results from this 1 image (200 iterations) don't give the results I want, what else can I do to increase the performance of the model? 
1. Should I do incremental training on a couple of other images?
2. Are there other parameters I can finetune instead of incremental training?

Note: I'm a novice.

**Comment by hansam95** [comment]
In this work, training is designed for a single style image per model, meaning you can only use one style image during training.

To achieve better results, you can try generating images during training to monitor progress and adjust the number of training iterations. Or you can adjust the learning rate for better performance.
